generic.js

```diff
@@ -2,4 +2,6 @@ var vehicle = "car";
 
 if(vehicle === "motorbike") {
   console.log("You should add a couple of wheels")
+} else if(vehicle ===    "car") {
```

Comment:
Codacy found an issue: [Multiple spaces found before '"car"'.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26445)
```suggestion
} else if(vehicle === "car") {
```